core/src/main/java/com/yahoo/imapnio/async/internal/ImapAsyncSessionImpl.java

```diff
@@ -208,7 +221,7 @@ private void sendRequest(@Nonnull final Object request) throws ImapAsyncClientEx
         }
 
         final ImapCommandEntry entry = requestsQueue.peek();
-        if (logger.isDebugEnabled()) {
+        if (isDebugEnabled()) {
             logger.debug(CLIENT_LOG_REC, sessionId, command.getLogLine());
```

Comment:
this will not work as logger.debug will only log if the logger.isdebugenabled is true, please test

Reply:
Done.